src/main/java/mate/academy/Main.java

```diff
@@ -23,11 +28,13 @@
         firstCinemaHall.setCapacity(100);
         firstCinemaHall.setDescription("first hall with capacity 100");
 
+        CinemaHallService cinemaHallService = (CinemaHallService) injector
+                .getInstance(CinemaHallService.class);
```

Comment:
Similar to the previous comment, ensure that the `injector.getInstance(CinemaHallService.class)` call returns an instance of `CinemaHallService` to avoid potential `ClassCastException`.